styles/05-objects/badge.css

```diff
@@ -0,0 +1,45 @@
+.badge-wrapper{
+  position: relative;
+  width: fit-content;
+  margin: 3ch;
+  display: inline;
+}
+
+.badge{
+  align-items: center;
+  border-radius: 50%;
+  display: inline-flex;
+  font-weight: 600;
+  justify-content: center;
+  padding: 5px;
+  position: absolute;
+}
+.badge-right{
+  right: -8px;
+}
+.badge-left{
+  left:-4px;
```

Comment:
Not sure if a -ve position is ok. The badge is outside of the icon. Could have some overlap.

Reply:
What would you suggest?

Reply:
Thanks for the suggestions. I'll think about them